plugins/source/snyk/client/transform.go

```diff
@@ -0,0 +1,16 @@
+package client
+
+import (
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Transform(tables schema.Tables) schema.Tables {
+	if err := transformers.TransformTables(tables); err != nil {
+		panic(err)
+	}
+	for _, t := range tables {
```

Comment:
We should be able to use `transformers.Apply` here so that it also gets applied to child tables https://github.com/cloudquery/plugin-sdk/blob/main/transformers/tables.go#L35

Reply:
Copied from digitalocean, tbh (cc: @disq )

Reply:
This was added recently to the SDK by @disq himself :)

Reply:
But the `schema.AddCQIDs` already handles relations, so that shouldn't be done twice IMO

Reply:
Hm okay, yeah then not... I guess 😅